src/components/ListToggleButton/__tests__/ListToggleButton.test.js

```diff
@@ -26,7 +27,7 @@ it('renders correctly in desktop and modal', () => {
   })
 
   const button = document.querySelector('button')
-  expect(button.textContent).toContain('Add to List')
+  expect(button.textContent).toContain(t({comment: "Test function", message: 'Add to List'}))
```

Comment:
If I can get the actual lingui working using a wrapper, should I translate the tests too? Or should I not try to get that working and just mock lingui?

Reply:
I would be very happy if the tests also included translation, so if you can get that working I am all for it!